src/OrbitQt/HistogramWidget.cpp

```diff
@@ -184,6 +191,8 @@ void HistogramWidget::paintEvent(QPaintEvent* /*event*/) {
 
   QPainter painter(this);
 
+  painter.fillRect(0, 0, Width(), Height(), kBackgroundColor);
```

Comment:
Why this is related with the PR?

Reply:
Because the point of this PR is to fix the colors used to plot histograms (including the background).

Reply:
Oh, sorry. I missed that part in the commit msg